keyboards/royal_kludge/rk61/rgb/rules.mk

```
@@ -0,0 +1,31 @@
+# project specific files
+SRC = ../../../drivers/led/sn32/matrix_sn32f24xx.c
+SRC += config_led.c
+
+# MCU name
+MCU = SN32F248BF
+
+# Build Options
+#   comment out to disable the options.
+#
+LTO_ENABLE = no
+BACKLIGHT_ENABLE = no
+MAGIC_ENABLE = yes
+MAGIC_KEYCODE_ENABLE = yes
+BOOTMAGIC_ENABLE = yes # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no    # Mouse keys
+EXTRAKEY_ENABLE = yes   # Audio control and System control
+CONSOLE_ENABLE = no     # Console for debug
+COMMAND_ENABLE = no     # Commands for debug and configuration
+SLEEP_LED_ENABLE = no   # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes       # USB Nkey Rollover
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = no
+SERIAL_LINK_ENABLE = no
+WAIT_FOR_USB = no
+CUSTOM_MATRIX = yes
+KEYBOARD_SHARED_EP = yes
```

```suggestion
KEYBOARD_SHARED_EP = no
```

I suggest having `KEYBOARD_SHARED_EP` disabled, having it enabled on some PCs may result to the keyboard being undetected during POST.